### PR TITLE
Added Support for Swift 4

### PR DESCRIPTION
**Pull request by calebkleveter**
This involved creating a `Package@swift-4.swift` file.

I also added the `Package.resolved` file to the `.gitignore`.

### CODE REVIEW

File: Package@swift-4.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import PackageDescription
+
+let package = Package(
+    name: "CSRF",
+    products: [
+        .library(name: "CSRF", targets: ["CSRF"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/vapor/vapor.git", from: "2.0.0"),
+        .package(url: "https://github.com/nodes-vapor/flash", from: "1.0.0"),
```
Comment (0xTim) [inline Package@swift-4.swift]: I don't think this dependency is needed?

Comment (calebkleveter) [reply Package@swift-4.swift]: I was unable to find where it was used in the code. Should I remove it?

Comment (0xTim) [reply Package@swift-4.swift]: Sounds good to me! And from the original Package.swift as well

Comment (calebkleveter) [reply Package@swift-4.swift]: Done.

**Pull request merged**